### PR TITLE
build(aio): upgrade deps

### PR DESCRIPTION
**Pull request by gkalpak**
## Size per file per commit

| File                                          | Initial  | cli@1.3.0    | core@4.3.4   | core@5.0.0-beta.3   | zone.js@0.8.16   | HttpClient   |
|-----------------------------------------------|---------:|-------------:|-------------:|--------------------:|-----------------:|-------------:|
| dist/0.b19e913fbdd6507d346b.chunk.js          |   14938  |   **14942**  |     14942    |            14942    |         14942    |     14942    |
| dist/inline.e07e02e29b7fc93816c6.bundle.js    |    1535  |      1535    |      1535    |             1535    |          1535    |      1535    |
| dist/main.f466098a873c1169a6dc.bundle.js      |  596429  |  **539996*** |  **542187**  |         **523703**  |        523703    |  **524385**  |
| dist/polyfills.9b05df3b6c9270ebf575.bundle.js |   34347  |   **34314**  |     34314    |            34314    |       **37402**  |     37402    |
| dist/worker-basic.min.js                      |   54001  |     54001    |     54001    |            54001    |         54001    |     54001    |
| _**Total**_                                   | _644788_ | _**644788**_ | _**646979**_ |        _**628495**_ |     _**631583**_ | _**632265**_ |

## Size-diff per file per commit

| File                                          | Initial  | cli@1.3.0    | core@4.3.4   | core@5.0.0-beta.3   | zone.js@0.8.16   | HttpClient   |
|-----------------------------------------------|---------:|-------------:|-------------:|--------------------:|-----------------:|-------------:|
| dist/0.b19e913fbdd6507d346b.chunk.js          |          |          +4  |              |                     |                  |              |
| dist/inline.e07e02e29b7fc93816c6.bundle.js    |          |              |              |                     |                  |              |
| dist/main.f466098a873c1169a6dc.bundle.js      |          |      -56433* |       +2191  |             -18484  |                  |        +682  |
| dist/polyfills.9b05df3b6c9270ebf575.bundle.js |          |         -33  |              |                     |           +3088  |              |
| dist/worker-basic.min.js                      |          |              |              |                     |                  |              |
| _**Total**_                                   |          |     _-56462_ |      _+2191_ |            _-18484_ |          _+3088_ |       _+682_ |

*: This value was before 088532bf, which brings in the latest buildOptimizer independently and is what causes the reduced size essentially.

### CODE REVIEW

**Comment by IgorMinar** [comment]
can you please update the limits for payload.sh to reflect the new numbers?

File: aio/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "ng": "yarn check-env && ng",
     "start": "yarn check-env && ng serve",
     "prebuild": "yarn check-env && yarn setup",
-    "build": "ng build --target=production --environment=stable -sm -bo",
+    "build": "ng build --target=production --environment=stable -sm -buildOptimizer",
```
Comment (IgorMinar) [inline aio/package.json]: please use the dash-case version `--build-optimizer`. everyone is kind of surprised that this pascalcase version works :)

**Review by IgorMinar — APPROVED** [review]
the rest looks fine. there are two example tests that are failing. do you know why?

File: aio/tools/examples/shared/yarn.lock [hunk 2]
```diff
@@ -5163,31 +5163,29 @@ request-progress@~2.0.1:
   dependencies:
```
Comment (gkalpak) [inline aio/tools/examples/shared/yarn.lock]: It was somehow updated as part of `yarn setup` or something and I thought it was harmless, but maybe not :grin:

**Comment by gkalpak** [comment]
<kbd>Comments addressed</kbd> + <kbd>CI green</kbd> <=> <kbd>Marking for merge</kbd>

**Pull request merged**